build_tools/requirements_conda.txt

```diff
@@ -7,3 +7,4 @@ nltk==3.2.1
 pytest==2.9.1       
 flask==0.12         # server
 requests==2.11.1
+matplotlib==1.5.3
```

Comment:
another error (which didn't arise at first though) is:
    UnsatisfiableError: The following specifications were found to be in conflict:
      - cython ==0.25.2
      - python 3.6*
    Use "conda info <package>" to see the dependencies for each package.

Looks like Python 3.6 and new packaging is not ok on Windows.